plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -77,41 +84,25 @@ class ServiceDestroyModal extends React.Component {
   }
 
   handleRightButtonClick() {
-    this.props.deleteItem(this.shouldForceUpdate());
-  }
-
-  getDestroyMessage() {
     const { service } = this.props;
-    let serviceName = "";
+    const serviceName = service ? service.getId() : "";
+    const inputFieldDestroyValue = this.state.inputFieldDestroy;
 
-    if (service) {
-      serviceName = service.getId();
+    if (inputFieldDestroyValue === serviceName) {
+      this.props.deleteItem(this.shouldForceUpdate());
     }
+  }
 
-    if (service instanceof Framework) {
-      return (
-        <p>
-          This will only
-          {" "}
-          {UserActions.DELETE}
-          {" "}
-          the package scheduler for
-          {" "}
-          <span className="emphasize">{serviceName}</span>
-          . Any tasks that were started by this scheduler will persist in the system. Are you sure you want to continue?
-        </p>
-      );
-    }
+  handleChangeInputFieldDestroy(e) {
+    const { service } = this.props;
+    const serviceName = service ? service.getId() : "";
+    const inputValue = e.target.value;
+    const comparison = inputValue === serviceName;
```

Comment:
Addressed. Here I need to assign to a variable because I need to change from false to true and vice versa so doing it directly on the setState doesn't work where I compare the values and at the same time try to use coercion.